src/crc16.rs

```diff
@@ -1,16 +1,31 @@
-#[cfg(feature = "std")]
-use std::hash::Hasher;
 #[cfg(not(feature = "std"))]
 use core::hash::Hasher;
+#[cfg(feature = "std")]
+use std::hash::Hasher;
 
 pub use util::make_table_crc16 as make_table;
 
 include!(concat!(env!("OUT_DIR"), "/crc16_constants.rs"));
 
+/// Structure that holds all of the important values for calculating a CRC
+///
+/// # Definitions
+///
+/// *table:* Holds the table values based on the supplied polynomial for the fast CRC calculations
+///
+/// *initial:* The initial inut value. AKA reflect_in
```

Comment:
nit: `input` instead of `inut`, and a few other occurrences in this pr

Reply:
A lot of the comments were copy/pasted between crc16/32/64

Reply:
do you want to keep it, or it's actually a typo?